test/unit/ui/marker.test.js

```diff
@@ -109,16 +239,5 @@ test('Marker', (t) => {
         t.end();
     });
 
-    t.test('marker\'s offset can be changed', (t) => {
```

Comment:
we still need to test that the marker's offset can be changed

Reply:
Gotcha, thank you for going through this!  I'm still getting used to working with flow and will be making the necessary changes and pushing shortly.